nixos/modules/virtualisation/docker-containers.nix

```
@@ -24,6 +24,26 @@ let
           '';
         };
 
+        devices = mkOption {
+          type = with types; listOf str;
+          default = [];
+          description = ''
+            List of Linux devices to directly expose to this container.
+            Use with caution; mappings can potentially give far more access than you perhaps intend!
+
+            There are also some mount options available
+            as a third field; please refer to the
+            <link xlink:href="https://docs.docker.com/engine/reference/commandline/run/#add-host-device-to-container---device">
+            docker engine documentation</link> for details.
+          '';
+          example = literalExample ''
+            [
+              "/dev/dvb:/dev/dvb"
```

nitpick: if you're mapping it to the same location, you don't have to mention the destination path. Might be nice to change this for completeness of documentation.

```suggestion
              "/dev/dvb"
              "/dev/sdc:/dev/xvdc"
```

---

Good call (that's something that I've never used due to volumes doing the total opposite of that!), committed in 4e64f35